docs/quickstart.md

```diff
@@ -254,12 +254,12 @@ Congratulations! You've created your first binary using TCMalloc.
 
 ## What's Next
 
-* Read our [overview](overview), if you haven't already. The overview covers
+* Read our [overview](overview.md), if you haven't already. The overview covers
```

Comment:
Are these links not working for you? They work for us. Are you viewing them via https://google.github.io/tcmalloc/quickstart.html ?

Reply:
I'm using github.com. (My company has github.io blocked, though we can request unblocking portions of it.)

Reply:
Since the .md will render correctly on the github pages version *and* on the main github.com source tree view, let's go with that one.